### try catch 구문 동작 원리 제대로 파헤치기
개발자가 예상하지 못한 에러를 처리하는 문법  
```js
// 정상적인 코드
if (error1) {

} else if (error2) {

}...
```
```js
try {
  // 정상적인 코드
} catch (error) {
  // 에러 처리
}
```

try 구문의 코드를 실행하다가 에러가 발생하면, try 코드를 멈추고 => catch로 error와 함께 바로 빠져나간다.  
에러가 여러개 생길 수 있다면 가장 먼저 생긴 에러부터 잡아준다.  
